aiven/client/cli.py

```diff
@@ -1533,7 +1551,7 @@ def service__cli(self):
                 raise
             raise argx.UserError("Executable '{}' is not available, cannot launch {} client".format(command, service_type))
 
-    def _build_influx_start_info(self, url):
+    def _build_influx_start_info(self, url: str) -> Tuple[str, List, Mapping]:
```

Comment:
I tried that at first, `mypy` doesn't like it. Because:
- the return value `params` will be later used in an concatenation expession in `self.service__cli()`, so it must be a `List`
- It can't be `List[str]` because it could contain `None` value. `info.hostname`, `info.port`, `info.username` could be `None`
- It can't be `List[Optional[str]]` because mypy doesn't allow `List[str] + List[Optional[str]]` :sweat: 

So I decided it keep it generic like this

Reply:
Ah, OK. Make sense. :+1: